test/suite_test.go

```diff
@@ -163,7 +163,11 @@ func resetWaitForStack() {
 
 // internalWaitForStackState refetches the given stack, until its .lastUpdated.state field matches
 // the one given. NB it fetches into the pointer given, so mutates the struct it's pointing at.
-func internalWaitForStackState(stack *pulumiv1.Stack, state shared.StackUpdateStateMessage) {
+func internalWaitForStackState(stack *pulumiv1.Stack, state shared.StackUpdateStateMessage, maybeTimeout ...string) {
```

Comment:
nit: `maybeTimeout` wasn't self-explanatory as a variable name to me initially. Perhaps `optionalTimeout` might be a better name?

Reply:
Sure. Different traditions :-)